lib/identity_cache/configuration_dsl.rb

```diff
@@ -279,7 +279,7 @@ def attribute_dynamic_fetcher(attribute, fields, values) #:nodoc:
       end
 
       def add_parent_expiry_hook(options)
-        child_class = options[:association_class]
+        child_class = options[:association_class].klass
```

Comment:
On Padrino with ActiveRecord the child class is not preloaded. I can fix this by adding `require "associatedmodel"` at the top of the file, but this is ugly. Maybe there's a way to alter this behaviour?